src/Commands/DevelopCommand.php

```diff
@@ -11,15 +11,19 @@ class DevelopCommand extends Command
 
     public function handle()
     {
+        // PHP_OS_FAMILY on Windows is "Windows"
+        $phpBinPackageDir = config('nativephp.binary_package');
```

Comment:
Why should this be configurable? 🤔

Reply:
Disregard as I just re-read your next review. Rolling it into a trait.